Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -51,8 +106,40 @@ public static int[] RemoveFirst(int[] array)
         /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveAt(int[] array, int index)
         {
-            // ToDo: implement.
-            return array;
+            if (array == null)
+            {
+                return array; 
+            }
+
+            var lengthOldArray = array.Length; 
+            
+            if (index < 0 || index == array.Length || array.Length == 0)
+            {
+                return array; 
+            }
+
+            var newArray = new int[lengthOldArray - 1];
+
+            for (int i = 0; i <= array.Length - 1 ; i++)
+			{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4887669)

Reply:
solved, no idea why Visual Studio does this